### DISCUSSION TITLE
BLTouch and "Z" light flash?

### DISCUSSION THREAD

**Discussion by JEBurroughs** [mriscoc/Ender3V2S1#346] [Q&A]
I have noticed with the firmware that if the Mesh Leveling box is checked that the "Z" offset icon at the right bottom of the screen flashes. I assumed this let you know that the Leveling was active. However today I happened to turn the LED lights I have attached off, and I noticed that the BLTouch light was flashing. Are they simply tied together, or does it indicate a different situation?

https://www.youtube.com/watch?v=BpJLbxoeuIQ

**Comment by mriscoc** [comment]
Hi, if the BLTouch blinks indicates that there was a problem, that could be as simple as the probe not having enough space to fully extend when the automatic test was performed at startup because it was too close to the bed.

**Reply by JEBurroughs** [reply]
I have print going on both now. The printer in the video is 7 hours into a 10 hour build. Looks beautiful, however the Z and Probe are still blinking. The other printer is also printer great and the Probe is solid Red and the "Z" is blinking so I seem to have created 2 different scenarios. I'll post back when they finish and do some testing. I will post my findings. I wonder if I have my BLTouch too low. It seems just right the tip is about 1mm from build plate when the probe triggers.

**Reply by JEBurroughs** [reply]
The flashing BLTouch light just took a power cycle to fix. Both just have the Z-Offset light that flashes while printing, and I think that is supposed too, so I think I'm good.

**Reply by mriscoc** [reply]
So, the cause was "...that could be as simple as the probe not having enough space to fully extend when the automatic test was performed at startup..."

**Reply by JEBurroughs** [reply]
I feel confident it was a probe glitch and nothing to worry about, it hasn't happened since. Thanks again for all your hard work for all of us!!!
